nx-X11/lib/Xcursor/AUTHORS

```diff
@@ -1,2 +0,0 @@
-Keith Packard, HP
-
```

Comment:
I still think `Xcursor` removal should be part of a unique PR (or this PR be extended to `Xcursor` in general — by also removing references in X11.templ and host.def)